common/components/chrome/MainHeader.jsx

```diff
@@ -113,13 +113,13 @@ class MainHeader extends React.Component<{||}, State > {
           <div className={this._cx.get('rightContent')}>
             {this._renderSectionLinks()}
             {this._renderHeaderLinks()}
+            {this._renderHeaderButtons()}
+            <div className="vertical-divider"></div>
             {
               CurrentUser.isLoggedIn() ?
                 this._renderAccountInfo() :
-                this._renderLogInButton()
```

Comment:
Remove the functions if we're not using them anymore.

Reply:
Addressed everything in the latest commit